index.html

```diff
@@ -256,21 +256,28 @@ <h1 class="contact-me-form-heading">
       </p>
       <form  action="https://formspree.io/f/mdovrdrq"
       method="POST" class="contact-me-form">
+        <label for="name"></label>
         <input
           class="contact-me-form-text-input"
           required
           maxlength="30"
```

Comment:
You are requested to use the min length option as well so that the required information could not be skipped.

Reply:
Okay, noted, i will add min-length